tasks/definitions/tap/start.js

```diff
@@ -3,6 +3,14 @@ const { launchBrowsers } = require('HerkinTasks/utils/playwright/launchBrowsers'
 const { setMountEnvs } = require('HerkinTasks/utils/envs/setMountEnvs')
 const { validateConfig } = require('HerkinTasks/utils/validation')
 
+/**
+ * @param {String} configPath - path to herkin config file
+ * @returns {String} path without config file
+ */
+const getRootPath = configPath => configPath
```

Comment:
Since we have the config path already, and the `rootDir` is defined in the config,
We should just load the config, and pull the `rootDir` value of there

This would allow the root to more configurable and not dependent on the `herkin.config` location

Reply:
Also, it would be good to also allow setting the config path via an ENV. 
Something like `HERKIN_CONFIG_PATH`, which would work exactly like the `--config` option

Reply:
UPdated 👍